clients/fides-js/rollup.config.mjs

```diff
@@ -8,14 +8,15 @@ import nodeResolve from "@rollup/plugin-node-resolve";
 import postcss from "rollup-plugin-postcss";
 import commonjs from "@rollup/plugin-commonjs";
 import { visualizer } from "rollup-plugin-visualizer";
+import strip from "@rollup/plugin-strip";
 
 const NAME = "fides";
 const IS_DEV = process.env.NODE_ENV === "development";
 const GZIP_SIZE_ERROR_KB = 45; // fail build if bundle size exceeds this
 const GZIP_SIZE_WARN_KB = 35; // log a warning if bundle size exceeds this
 
 // TCF
-const GZIP_SIZE_TCF_ERROR_KB = 86;
+const GZIP_SIZE_TCF_ERROR_KB = 85;
```

Comment:
this was recently bumped up, but we can bump in back down now that we're stripping loggers from the prod code.